### PR TITLE
reword the intro to focus less on cost of sensors

### PR DESCRIPTION
**Pull request by nick-geofroy**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -78,9 +78,9 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{INTRODUCTION}
 
-In the advent of connected vehicles and autonomous transportation, equipping consumer vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems can be prohibitively expensive.
-Technologies such as Radar, LIDAR and high-end cameras that are typically used for collision prediction are likely too expensive for most consumers.
-On the other hand, dash-cams, which are cheap video cameras that can be mounted to a car's dashboard, have become increasingly affordable and prevalent as a means to capture a driver’s perspective.
+In the advent of connected vehicles and autonomous transportation, we are equipping vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems.
```
Comment (Cvarier) [inline paper.tex:81]: Change this to "In the advent of connected vehicles and autonomous transportation, it has become commonplace to equip vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems."

Comment (cvarier) [inline paper.tex]: Change this to "In the advent of connected vehicles and autonomous transportation, it has become commonplace to equip vehicles with the necessary equipment to gather real-time data and facilitate high-end automated collision avoidance systems."

File: paper.tex [hunk 2]
```diff
@@ -78,9 +78,9 @@
...
+Using these vehicles to simulate collision situations has a two-fold cost. Both the car and expensive / experimental sensors are put at risk.
```
Comment (cvarier) [inline paper.tex]: I would make the following change:

"Using these vehicles to simulate collision situations has a two-fold cost: both the car and expensive data-gathering equipment are put at risk."

File: paper.tex [hunk 3]
```diff
@@ -78,9 +78,9 @@
...
+However, we can collect collision data naturally in real driving scenarios through the use of dash-cams, cheap video cameras that can be mounted to a car's dashboard.
```
Comment (cvarier) [inline paper.tex]: I would change the last comma to a hyphen:

"However, we can collect collision data naturally in real driving scenarios through the use of dash-cams - cheap video cameras that can be mounted to a car's dashboard."

**Review by cvarier — CHANGES_REQUESTED** [review]

**Pull request merged**